guides/v2.1/config-guide/bootstrap/magento-bootstrap.md

```diff
@@ -98,8 +98,8 @@ We have the following entry point applications (that is, applications defined by
 In default or developer mode, a request for a non-existent static resource is redirected to the static entry point according to the rewrite rules specified by the appropriate `.htaccess`.
 When the request is redirected to the entry point, the Magento application parses the requested {% glossarytooltip a05c59d3-77b9-47d0-92a1-2cbffe3f8622 %}URL{% endglossarytooltip %} based on retrieved parameters and finds the requested resource.
 
-*	In developer mode, the content of the file is returned so that every time the resource is requested, the returned content is up to date.
-*	In <a href="{{ page.baseurl }}/config-guide/bootstrap/magento-modes.html#default-mode">default</a> mode, the retrieved resource is published so it is accessible by the previously requested URL.
+*	In <a href="{{ page.baseurl }}/config-guide/bootstrap/magento-modes.html#developer-mode">developer mode</a>, the content of the file is returned so that every time the resource is requested, the returned content is up to date.
```

Comment:
@RakeshJesadiya You can not add HTML tag in markdown, Please update it.

Reply:
You **could** use HTML here, as you have, because it's not within an HTML note. But we use markdown for links (when not in an HTML note or HTML table). Please change this link to a markdown coded link.